features/telemetry/protocols/flows/src/main/java/org/opennms/netmgt/telemetry/protocols/flows/AbstractFlowAdapter.java

```diff
@@ -116,6 +116,10 @@ public void handleMessageLog(TelemetryMessageLog messageLog) {
             }
         } catch (FlowException ex) {
             LOG.error("Error while persisting flows: {}", ex.getMessage(), ex);
+            if (ex.getMessage().equals("Connection pool shut down")) {
```

Comment:
Is there a better way to catch this, or match the exception? Seems like this could break easily.

Reply:
Added a dedicated exception to handle this more properly.